### DISCUSSION TITLE
Rnode network segregation

### DISCUSSION THREAD

**Discussion by Heavyclouds** [markqvist/Reticulum#374] [Help & Questions]
I know that an Rnode is designed to mesh by default. However I am trying to create Rnode pairs that do not mesh. I have reflashed my boards with a different freq, tried changing the freq, spreading factor, and coding rate in the .reticulum/config file and my Rnodes still see each other. Is there some freq config I am missing? Or is this even possible? Thanks!

**Comment by faragher** [comment]
Your situation seems unclear. Allow me to answer for both cases.

If you have two machines with RNodes with different settings, they should not communicate. You likely either have AutoInterface on in the config (linking them over your local network) or have an Interface in common (such as a TCP Client interface to the TestNet or an I2P interface). If neither of these are the case, then I'm uncertain the problem and would like to ask for your config file.

Alternately, you have one machine with two RNodes. In this case, they will bridge if enable_transport is true. I don't believe it will bridge the two as a default.

As an edge case, if you have one machine that you want to route two isolated networks, you'd have to disable share_instance and run RNS twice with two different config files. Given the price of SBCs, the ability to run VMs, and the low overhead of a docker container, I'd suggest not trying to run two parallel RNS instances, but I doubt that's your use case.

**Reply by cobraPA** [reply]
Agreed, and curious how you are testing that they 'see each other' @Heavyclouds ?  Can you give more detail of your setup?
Note you do not have to 'flash' the settings you are mentioning, they are usually set by Reticulum on starting the RNode (there are some limits on what frequency can be changed to based on the hardware used and region), so maybe they are just being set on boot and not using your changes?
